.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: developmentseed/circleci:node8.4-python2.7
```

Comment:
Any specific reason you choosed this image over the 'official' circleci node image (https://hub.docker.com/r/circleci/node/)? Is that for python?
I think using image 
`- image: circleci/node:8` or `- image: circleci/node:8.9` would be a good call.
Please note that v8.4 is  actually not LTS (8.9.0 is).